api/config/rolepermission.php

```diff
@@ -675,6 +746,9 @@
             'teamRole' => [
                 'any' => ['assign'],
             ],
+            'assessmentPlan' => [
+                'any' => ['view'],
+            ]
```

Comment:
Some of these will need changes in the router file too, like addition here so that community managers may view the assessment plan. 

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/dda71474-82be-4f38-b2a3-b6835678423a)

Reply:
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/befc949f-afd7-4221-87b5-40cc711e31d7)